src/main/java/com/google/sps/data/FakePersonDao.java

```diff
@@ -14,39 +14,58 @@
 
 package com.google.sps.data;
 
+import java.util.ArrayList;
 import java.util.HashMap;
+import java.util.List;
 import java.util.Optional;
+import java.util.Random;
 
 /** Mimics accessing Datastore to support managing Person entities. */
 public class FakePersonDao implements PersonDao {
-  // @param datastore the fake database.
-  private HashMap<String, Person> datastore;
+  private HashMap<Long, Person> storedObjects;
 
   /** Initializes the fields for PersonDatastoreDAO. */
   public FakePersonDao() {
-    datastore = new HashMap<String, Person>();
+    storedObjects = new HashMap<Long, Person>();
   }
 
-  /** We put person into datastore with email as the key. */
+  /** We put person into storedObjects with a randomly generated long as its id. */
   @Override
   public void create(Person person) {
-    datastore.put(person.email(), person);
+    long id = new Random().nextLong();
```

Comment:
I see that UserService's User type has as "getUserId".

https://cloud.google.com/appengine/docs/standard/java/javadoc/com/google/appengine/api/users/User.html#getUserId--

Does it make sense to use this ID instead of our own ID? This would eliminate the need to look up the user by email on every call.

Reply:
Done